dist/faq.html

```diff
@@ -56,7 +56,7 @@
         <div class="question">
             <h4>What is the scoring methodology based on?</h4>
 
-            <p>It is extremely difficult to assign an objective value to a subjective question such as "How bad is not implementing HTTP Strict Transport Security?"  This is complicated by the fact that what may be unnecessary for for one site &mdash; such as implementing Content Security Policy (CSP) &mdash; may be an absolute requirement for a different site.</p>
+            <p>It is extremely difficult to assign an objective value to a subjective question such as "How bad is not implementing HTTP Strict Transport Security?"  This is complicated by the fact that what may be unnecessary for for one site &mdash; such as implementing Content Security Policy (CSP) &mdash; might mitigate important risks for a different site.</p>
             <p>The scores and grades offered by the Mozilla Observatory are designed to alert developers when they're not taking advantage of the latest web security features, as recommended in the Mozilla <a href="https://wiki.mozilla.org/Security/Guidelines/Web_Security">web security guidelines</a> and <a href="https://wiki.mozilla.org/Security/Server_Side_TLS">server side TLS guidelines</a>.  Individual developers will need to determine which of these security technologies is right for their sites.</p>
         </div>
```

Comment:
You should make at least two more points:

1. This is not a vulnerability scanner. Thus, for instance, if you have two sites, one with a big XSS vulnerability and CSP and one with no XSS vulnerability, it rates the first one higher even though it's objectively less secure.

2. The weights will probably be adjusted in the future as stuff evolves.